release_readme.rst

```diff
@@ -28,15 +28,17 @@ Package Contents
 Overview
 --------
 
-The F5® OpenStack LBaaSv1 plugin allows you to orchestrate BIG-IP® load balancing services – including virtual IPs, pools, device service groups, and health monitoring – in an OpenStack environment.
+The F5 OpenStack LBaaSv1 plugin allows you to orchestrate BIG-IP® Local
+Traffic Manager™ (LTM®) services – including virtual IPs, pools, device service groups, and health monitoring – in an OpenStack environment.
```

Comment:
Is the linebreak intentional to split up "... Local" from "Traffic Manager ..."?